### PR TITLE
@eessex => Fullscreen updates

### PR DESCRIPTION
**Pull request by kanaabe**
Closes https://github.com/artsy/force/issues/1530
fullscreenheader

fullscreenheader2

### CODE REVIEW

**Comment by kanaabe** [comment]
Sorry, just need to address one more thing with the share nav, please hold!

**Comment by kanaabe** [comment]
Addressed those comments in the issue -- show the author/date/time on SA and hide share nav until the FS header is out of view:

image

File: desktop/components/article/client/fullscreen.coffee [hunk 1]
```diff
@@ -3,59 +3,25 @@ Backbone = require 'backbone'
...
 
-  events:
-    'click .article-fullscreen-down-arrow a': 'scrollPastFullscreenHeader'
-
   initialize: (options) ->
     { @article, header } = options
     @$window = $(window)
     @setupArticleWaypoints()
-    @initFullscreenHeader(header)
     @duration = 500
```
Comment (eessex) [inline desktop/components/article/client/fullscreen.coffee]: is the `@duration` still used here?

File: desktop/components/article/client/fullscreen.coffee [hunk 2]
```diff
@@ -3,59 +3,25 @@ Backbone = require 'backbone'
...
-      @$('.article-fullscreen, .article-fullscreen-overlay, .article-fullscreen-video-player, .article-fullscreen-image').css 'min-height', headerHeight
-
-    $container.css 'margin-top': "#{margin}px"
-
-  scrollPastFullscreenHeader: ->
-    position = @$('.article-fullscreen').height()
-    (@$htmlBody ?= $('html, body'))
-      .animate { scrollTop: position }, @duration
-    false
-
```
Comment (eessex) [inline desktop/components/article/client/fullscreen.coffee]: nice diff!

File: desktop/components/article/stylesheets/fullscreen_article.styl [hunk 3]
```diff
@@ -32,128 +24,126 @@ body.body-fullscreen-article
...
+    justify-content space-between
+    align-items flex-start
+  &__text, &__share
+    padding-bottom 50px
+  &__container
+    width 100%
+    height 100%
+    display flex
+    flex-direction column
+    justify-content space-between
```
Comment (eessex) [inline desktop/components/article/stylesheets/fullscreen_article.styl]: hooray for bem

**Comment by eessex** [comment]
This looks good to me, just rebuilt since the x tests were unrelated.  Feel free to merge when 💚

File: desktop/components/article/client/fullscreen.coffee (hunk 1 above)
Comment (kanaabe) [inline desktop/components/article/client/fullscreen.coffee]: 👍  Good catch, removed!

**Pull request merged**